cmd/kops/update_cluster.go

```diff
@@ -66,7 +66,10 @@ type UpdateClusterOptions struct {
 	OutDir          string
 	SSHPublicKey    string
 	RunTasksOptions fi.RunTasksOptions
-	CreateKubecfg   bool
+
+	CreateKubecfg bool
+	admin         bool
+	user          string
```

Comment:
These two should probably be public.

Reply:
I see these are made public a bit inconsistently. In my opinion, probably all of these should not be exposed (like in create_cluster). I just prefer to leave them unexposed and then unexpose the existing flag at some later time.

Reply:
Different issue, but some of the logic in create_cluster needs to be moved into pkg so it can be called as imported code. I haven't gone through update_cluster to see if there's logic that should be exposed in pkg.

Reply:
There's only one block of logic in `RunUpdateCluster` that should be pushed down into `ApplyClusterCmd`, so consider this resolved.